src/stx/common/trex_rx_timesync.cpp

```diff
@@ -91,6 +91,17 @@ TimesyncPacketParser_err_t RXTimesync::parse_ptp_pkt(uint8_t *pkt, uint16_t len,
         parse_delay_response(delay_resp, &t);
         m_timesync_engine->receivedPTPDelayResp(port, *(header->seq_id), t, header->source_port_id,
                                                 delay_resp->req_clock_identity);
+        if (hardware_timestamping_enabled && m_timesync_engine->isDeltaValid(port)) {
```

Comment:
I think this operation fits better to trex_timesync file, trex_rx_timesync looks more like rx packets parsing file and all "delta" operations are in trex_timesync